Adamant/Modules/ChatsList/ChatListViewController.swift

```diff
@@ -1106,7 +1109,12 @@ extension ChatListViewController {
         
         return UISwipeActionsConfiguration(actions: actions)
     }
-    
+    func tableView(_ tableView: UITableView, didEndEditingRowAt indexPath: IndexPath?) {
+        swipedIndex = nil
```

Comment:
does this method get called on regular `reloadData` call? Or do we need to reset `swipedIndex` before each `reloadData` call?